README.md

```diff
@@ -4,13 +4,14 @@ Testnets for [Regen Ledger](https://github.com/regen-network/regen-ledger)
 
 ## Active Testnets
 
-### regen-testnet-2
+### regen-redwood-1
 
-`regen-testnet-2` is active now and here are some important details:
+`regen-redwood-1` is active now and here are some important details:
```

Comment:
```suggestion
`regen-redwood-1` is an active testnet for the community. It generally runs the same software as mainnet and useful for testing third-party integrations. Also it will be used as a platform for testing mainnet upgrades.
```